scss/abstracts/_variables.scss

```diff
@@ -42,4 +42,17 @@ $gold--90: #472214;
 $gold--70: #E05A00;
 $gold--50: #FF871F;
 $gold--30: #FFA947;
-$gold--10: #FFE7B8;
\ No newline at end of file
+$gold--10: #FFE7B8;
+
+
+$device--sm: 65em;
+// $device--md: 60em;
+$device--lg: 82em;
+$actions-width: 7.25rem;
+$header-button-height: 2.75rem;
```

Comment:
The `actions-width` and `header-button-height`, are those things we should move into their own scss file? Only because this file is likely to imported in all consumers of this lib.

Reply:
yup good call!